README.md

```diff
@@ -93,12 +93,16 @@ require('alexa-app-server').start({
 	// root of your web server.
 	app_root: "/alexa/",
 
-	// The directory containing server-side processing modules (see below)
+	// The directory containing server-side processing modules (see below).
 	server_dir: "server",
 
-	// The port the server should bind to
+	// The port the server should bind to.
 	port: 80,
 
+	// The host address in which the server should bind to. If not specified,
+	// this argument will be ignored.
+	host_address: undefined,
```

Comment:
I guess one thing to note is that is it a breaking change to some extent. If we look at the node docs for `.listen` [here](https://nodejs.org/api/http.html#http_server_listen_port_hostname_backlog_callback) (express's `app.listen` does call `http.listen`), when you don't specify a hostname, it will try to find any IPv4/IPv6 address.

What we could do is either add a note about the recommended way to setup the server bindings or maybe deprecate the old style method in favor of the new one.